pkg/terraform/terraform_test.go

```diff
@@ -130,16 +130,16 @@ func TestTerraform(t *testing.T) {
 }
 
 // Create a Terraform client, pointing at a temp dir
-func terraformTemp(hcl string) (*Terraform, func() error /*cleanup*/, error) {
+func terraformTemp(hcl string) (*Terraform, func() /*cleanup*/, error) {
```

Comment:
don't return errors from `os.RemoveAll` in tests, so we don't need to handle them everywhere.